core/server/master/src/main/java/alluxio/master/AlluxioSecondaryMaster.java

```diff
@@ -24,16 +24,18 @@
 import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
+import java.util.concurrent.CountDownLatch;
 
 import javax.annotation.concurrent.NotThreadSafe;
 
 /**
  * The secondary Alluxio master that replays journal logs and writes checkpoints.
  */
 @NotThreadSafe
-public final class AlluxioSecondaryMaster implements Server {
+public final class AlluxioSecondaryMaster implements Process {
   private static final Logger LOG = LoggerFactory.getLogger(AlluxioSecondaryMaster.class);
-  private MasterRegistry mRegistry;
+  private final MasterRegistry mRegistry;
+  private final CountDownLatch mLatch;
```

Comment:
only two out of four processes use it, so I am not sure what would be a good abstraction here ... any suggestions?